locales/connection/active.en.toml

```
@@ -2,7 +2,7 @@
 one = 'not logged in. Run "rhoas login" to authenticate'
 
 [connection.error.notLoggedInMASError]
-one = 'not logged in to MAS-SSO. Run "rhoas login" to authenticate'
+one = 'not logged in to MAS-SSO. Run "rhoas login" to authenticate. Note: token-based login is not supported by the Kafka admin commands.'
```

It is not clear what are kafka admin commands. As user I would go and look `rhoas kafka admin` but we do mix admin commands with control plane ones. It feels that we need to list them to make it more clear. 


```suggestion
one = 'not logged in to MAS-SSO. Run "rhoas login" to authenticate. Note: token-based login is not supported by the Kafka topics and consumergroup commands.'
```